src/Components/Milestones.js

```diff
@@ -22,7 +22,7 @@ function Milestones({ milestones }) {
   )
 
   const content = (milestone) => (
-    <Card title={milestone.title} subTitle={milestone.date} className="p-m-5">
+    <Card title={milestone.title} subTitle={milestone.date} className="p-m-5 p-shadow-15">
```

Comment:
Looks good 👍 , there are other changes in the PR?

Reply:
no, there are no more changes in the PR for now, I can further add the changes you had like to suggest.